gearman-worker.sh

```diff
@@ -4,12 +4,19 @@
 
 # Get a Bash array containing a list of all the job names.
 if [ -z ${GEARMAN_FUNCTIONS:+set} ]; then
-  GEARMAN_FUNCTIONS=(`$DRUSH --root=$DRUPAL_ROOT php-eval --format=string "return implode('$IFS', array_keys(module_invoke_all('islandora_job_register_jobs')));"`)
-  DRUSH_RETURN=$?
-  if [ $DRUSH_RETURN -ne 0 ]; then
-    logger "Could not determine worker functions."
-    exit 1
-  fi
+  GEARMAN_FUNCTIONS=""
+  # XXX: STDOUT from Drush commands using @sites comes with an unfortunate
+  # amount of unfriendly baggage; better to loop manually.
+  for site in $SITE_URI_LIST; do
+    GEARMAN_FUNCTIONS="${GEARMAN_FUNCTIONS} `$DRUSH --root=$DRUPAL_ROOT --uri=$site islandora-job-list-functions --param_format`"
```

Comment:
And then append to the array:
```bash
GEARMAN_FUNCTIONS+=(`$DRUSH --root=$DRUPAL_ROOT --uri=$site islandora-job-list-functions`)
```
... I think? Could then allow the replacement below, with the `-f ` prefixing?